### DISCUSSION TITLE
Extra spaces at the end of each line

### DISCUSSION THREAD

**Discussion by EyadAbuArqoub** [massCodeIO/massCode#48] [General]
Hello massCode,

Thanks for the new version 2.2.0

I have noted that each line ends with extra spaces, and when we delete it the text deleted also.

![image](https://user-images.githubusercontent.com/17927572/164136193-cf2d8f25-dd99-4695-8be9-f75873dd1bb9.png)

Thanks.

**Comment by EyadAbuArqoub** [comment]
Regarding to the view, I think v1.3.1 was better.

**Reply by antonreshetov** [reply]
@EyadAbuArqoub This is most likely due to the specifics of Ace. Preferences -> Editor -> Show Invisibles and you will see the real thing.

> Regarding to the view, I think v1.3.1 was better. 

You may not use it at all.
The current look is an imitation of the native app on macOS in the first place.
